main/acle.rst

```diff
@@ -4521,306 +3667,152 @@ each architecture includes its predecessor instruction set.
+| UMAAL           |           | 6, 7E-M       | C                                                                                           |
+| UMLAL           |           | all, 7-M      | `acc += (uint64_t)x * y`                                     |
+| UMULL           |           | all, 7-M      | C                                                                                           |
+| UQADD16         |           | 6, 7E-M       | `__uqadd16`                                                  |
+| UQADD8          |           | 6, 7E-M       | `__uqadd8`                                                   |
+| UQASX           |           | 6, 7E-M       | `__uqasx`                                                    |
+| UQSAX           |           | 6, 7E-M       | `__uqsax`                                                    |
+| UQSUB16         |           | 6, 7E-M       | `__uqsub16`                                                  |
+| UQSUB8          |           | 6, 7E-M       | `__uqsub8`                                                   |
+| USAD8           |           | 6, 7E-M       | `__usad8`                                                    |
+| USADA8          |           | 6, 7E-M       | `__usad8 + acc`                                              |
+| USAT            | Q         | 6, 7-M        | `__usat`                                                     |
+| USAT16          | Q         | 6, 7E-M       | `__usat16`                                                   |
+| USAX            |           | 6, 7E-M       | `__usax`                                                     |
+| USUB16          |           | 6, 7E-M       | `__usub16`                                                   |
+| USUB8           |           | 6, 7E-M       | `__usub8`                                                    |
+| UXTAB           |           | 6, 7E-M       | `(uint8_t)x + i`                                              |
+| UXTAB16         |           | 6, 7E-M       | `__uxtab16`                                                  |
+| UXTAH           |           | 6, 7E-M       | `(uint16_t)x + i`                                             |
+| UXTB16          |           | 6, 7E-M       | `__uxtb16`                                                   |
+| UXTH            |           | 8,6, 6-M      | `(uint16_t)x`                                                 |
+| VFMA            |           | VFPv4         | `fma`, `__fma`                  |
+| VSQRT           |           | VFP           | `sqrt`, `__sqrt`                |
+| WFE             |           | 8,6K, 6-M     | `__wfe`                                                      |
+| WFI             |           | 8,6K, 6-M     | `__wfi`                                                      |
+| YIELD           |           | 8,6K, 6-M     | `__yield`                                                    |
```

Comment:
Nit: please fix the width of the rightmost column (you can remove a lot of white space, and align all the |)